demo/management/commands/clean_stale_db.py

```diff
@@ -59,7 +59,7 @@ def drop_all_stale_databases(force=False, max_days=3, *args, **kwargs):
             if dropped:
                 deleted_databases.append(database.name)
                 database.delete()
-    reflect_db_objects(get_empty_metadata())
+                reflect_db_objects(get_empty_metadata(), db_name=database.name)
```

Comment:
Before we were reflecting Django models for every database, irrespective of what the clean up did. Now we'll be reflecting only the databases we affected.